docs/GETTING_STARTED.md

```diff
@@ -284,13 +309,116 @@ const i18nState = factoryI18nextState({
 export default i18nState
 ```
 
-Description of options for `factoryI18nState`.
+## Integrating i18next [remote]
 
-|    Property    |            Type            |      default       | Description                                                                                                                                                    |
-| :------------: | :------------------------: | :----------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| clearNamespace | `Function(String): String` | :heavy_check_mark: | The default function strips out initials `src/`, `DEV/` or `dist/.app`. Also, strips extensions (`.js`, `.jsx`, `.svelte`) and converts remaining `/` into `.` |
-|      i18n      |         `i18next`          |                    | It is required, and it is the mechanism to resolve messages                                                                                                    |
-|  persistence   |          `Object`          | :heavy_check_mark: | _(`default: { get: noop, set: noop }`)_ The service to get and persist locale changes                                                                          |
+From static usage to remote (fetching resources on demand), the `i18next` the library will need to have the property `load: 'currentOnly'` as well as the load path.
```

Comment:
the load option depends on how the app is localized... could be also an other option, but ok